docs/reference/release-notes/8.0.0.asciidoc

```diff
@@ -0,0 +1,699 @@
+[[release-notes-8.0.0]]
+== {es} version 8.0.0
+
+coming::[8.0.0]
+
+The following list are changes in 8.0.0 as compared to 7.17.0, and combines
+release notes from the 8.0.0-alpha1, -alpha2, -beta1, -rc1 and -rc2 releases.
+
+Also see <<breaking-changes-8.0,Breaking changes in 8.0>>.
+
```

Comment:
```suggestion
[[known-issues-8.0.0]]
[float]
=== Known issues

* If you installed {es} from an archive on an aarch64 platform like Linux ARM or macOS M1, the
`elastic` user password and {kib} enrollment token are not generated
automatically when starting your node for the first time.
+
--
After the node starts, generate the `elastic` password with the
<<reset-password,`bin/elasticsearch-reset-password`>> tool:

[source,bash]
----
bin/elasticsearch-reset-password -u elastic
----

Then, create an enrollment token for {kib} with the
<<create-enrollment-token,`bin/elasticsearch-create-enrollment-token`>> tool:

[source,bash]
----
bin/elasticsearch-create-enrollment-token -s kibana
----
--
```

This is https://github.com/elastic/elasticsearch/pull/83654

Reply:
Thanks. Added in https://github.com/elastic/elasticsearch/pull/83689/commits/b4fbb7ace5e3ebf3fe1a3b9f1685d7d5815a160c